sdk/search/ai-search/README.md

```diff
@@ -0,0 +1,302 @@
+# Azure Cognitive Search client library for JavaScript
+
+[Azure Cognitive Search](https://docs.microsoft.com/en-us/azure/search/) is a search-as-a-service cloud solution that gives developers APIs and tools for adding a rich search experience over private, heterogeneous content in web, mobile, and enterprise applications.
+
+Azure Cognitive Search is well suited for the following application scenarios:
+
+- Consolidation of heterogeneous content types into a private, single, searchable index. Queries are always over an index that you create and load with documents, and the index always resides in the cloud on your Azure Cognitive Search service. You can populate an index with streams of JSON documents from any source or platform. Alternatively, for content sourced on Azure, you can use an indexer to pull data into an index. Index definition and management/ownership is a key reason for using Azure Cognitive Search.
+- Raw content is large undifferentiated text, image files, or application files such as Office content types on an Azure data source such as Azure Blob storage or Cosmos DB. You can apply cognitive skills during indexing to add structure or extract meaning from image and application files.
```

Comment:
What is `undifferentiated` in this context? Like...unstructured?

Reply:
Yeah, you can literally feed it a PDF of a newsletter and it will try to parse out stuff you can search (like OCRing text or calling out you have a picture of Bill Gates)